lib/requirefy.js

```diff
@@ -17,17 +17,29 @@ module.exports = function requirefy(code, options, resolvePath) {
     return header;
   }
 
-  function removeRequirejs(code, head, tail) {
+  function removeRequirejs(code, head, tail, moveUseStrict, useStricRange) {
     var top = code.substring(0, head.start - 1);
     var mid = code.substring(head.end, tail.start);
     var bot = code.substring(tail.end);
 
+    if (moveUseStrict) {
+      var beforeUseStrict = mid.substring(0, useStricRange[0]);
```

Comment:
I changed the range thingy to an object instead of Array, btw. More readable IMO